Assets/Scripts/GridSystem.cs

```diff
@@ -53,10 +53,11 @@ public GridSystem(int width, int height, float cellSize, Vector3 origin, Func<Gr
                 {
                     
```

Comment:
probably won't need the commented out code anymore

Reply:
id like to keep it for debugging later anyway, it doesnt need to be commented out anyway since showdebug is false, it wont go into that loop